applications/asset_tracker_v2/sample.yaml

```diff
@@ -10,6 +10,15 @@ tests:
       - thingy91_nrf9160_ns
       - native_posix
     tags: ci_build
+  applications.asset_tracker_v2.nrf_cloud.sysbuild:
+    build_only: true
+    sysbuild: true
+    build_on_all: true
+    platform_allow: nrf9160dk_nrf9160_ns thingy91_nrf9160_ns
+    integration_platforms:
+      - nrf9160dk_nrf9160_ns
+      - thingy91_nrf9160_ns
+    tags: ci_build sysbuild
```

Comment:
Is this going to be the first time that sysbuild is in NCS CI?

Or do sysbuild tags not run in CI?

Reply:
Sysbuild has been in CI for quite some time in samples/bluetooth/fast_pair/input_device, samples/bluetooth/peripheral_uart, samples/multicore/hello_world, and samples/zigbee/shell

Reply:
ah, I was not aware.